doc/refman/Universes.tex

```diff
@@ -131,6 +131,52 @@
   polymorphically, not at a single instance.
 \end{itemize}
 
+\asection{{\tt Cumulative, NonCumulative}}
+\comindex{Cumulative}
+\comindex{NonCumulative}
+\optindex{Inductive Cumulativity}
+
+Inductive types, coinductive types, variants and records can be
+declared cumulative using the \texttt{Cumulative}. Alternatively,
+there is an option \texttt{Set Inductive Cumulativity} which when set,
```

Comment:
This seems correct to me.  By contrast, if polymorphism is not set and you do `Cumulative Inductive `, that should be an error.